app/views/components/goodsLocationExpander.scala.html

```diff
@@ -34,106 +35,94 @@
 @expanderContent = {
     @paragraphBody(text("intro01"))
 
-    @paragraphBody(text("paragraph01.title"), "govuk-label--s")
-    @paragraph(link(text = text("paragraph01.link"), call = Call("GET", appConfig.rollOnRollOffPorts), target = "_blank"))
-
-    @paragraphBody(text("paragraph02.title"), "govuk-label--s")
-    @paragraph(link(text = text("paragraph02.link"), call = Call("GET", appConfig.railLocationCodes), target = "_blank"))
-
-    @paragraphBody(text("intro03"))
-
-    @paragraphBody(text("paragraph03.title"), "govuk-label--s")
+    @heading(text("paragraph01.title"), "govuk-heading-s govuk-!-margin-top-4 govuk-!-margin-bottom-1", "h2")
     @govukHint(Hint(
-        content = HtmlContent(text("paragraph03.text",
-            link(
-                text = text("paragraph03.link"),
-                call = Call("GET", appConfig.customsDecCompletionRequirements),
-                target = "_blank"
-            )
+        content = HtmlContent(link(
+            text = text("paragraph01.link"),
+            call = Call("GET", appConfig.rollOnRollOffPorts),
+            target = "_blank"
         ))
     ))
 
-    @paragraphBody(text("paragraph04.title"), "govuk-label--s")
+    @heading(text("paragraph02.title"), "govuk-heading-s govuk-!-margin-top-4 govuk-!-margin-bottom-1", "h2")
```

Comment:
Just wondering why we need all these styles for these headings (just never seen them used anywhere else for headings so slightly curios) :thinking:

Reply:
The content looked a bit weird with just the heading and a link:
![Screenshot from 2021-09-27 12-00-03](https://user-images.githubusercontent.com/2183310/134908588-aa20beca-6093-42b2-910b-08fcdccc5bc7.png)
So Liz asked if I could play about with the spacing a bit:
![Screenshot from 2021-09-27 12-55-49](https://user-images.githubusercontent.com/2183310/134908683-252377c2-1147-4b23-a2de-7919525e2da9.png)

Not usually needed but in this case I think that was just down to the sparseness of the content that it needed tweeking.